### ISSUE TITLE
Long press file name

### ISSUE THREAD

**Issue by mrahmadt** [nextcloud/ios#664]
When you press long a file with OneDrive it will allow you to drag and drop it to another folder

Not something important but frankly very nice user experience 

![Image](https://i.imgur.com/Mc581gR.jpg)

**Comment by rakekniven** [comment]
How does it work for larger folder structures?
How to navigate throught it?

**Comment by mrahmadt** [comment]
Drag the file to the top/bottom of the screen and it will scroll the list up/down

**Comment by rakekniven** [comment]
Yes, but how to navigate through a structure (many folder levels)?

**Comment by mrahmadt** [comment]
Drag file over folder name and it will open the folder or drag the file to the icon "Files" in the bar and will open the parent folder.
